.github/workflows/ci.yml

```diff
@@ -0,0 +1,103 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
```

Comment:
Is there a way to make this bit a bit clearer? At first I thought this meant you're *ignoring* PRs, but apparently it means all PRs.